common/djangoapps/course_modes/views.py

```diff
@@ -155,6 +156,10 @@ def get(self, request, course_id, error=None):
                 for x in verified_mode.suggested_prices.split(",")
                 if x.strip()
             ]
+
+            default_currency = settings.PAID_COURSE_REGISTRATION_CURRENCY or ['usd', '$']
```

Comment:
Please don't provide defaults for settings in application code like this. Any defaults should be specified in settings. If the setting doesn't have a value by this point, let the code raise an exception. Silently providing defaults like this makes it difficult to configure the system.